js/foam/u2/md/ToolbarAction.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.u2.md',
+  name: 'ToolbarAction',
```

Comment:
In U1 it was needed to associate actions that are being sent to a distant toolbar with the data they were supposed to operate on.

In U2, it probably won't be needed soon, based on the toolbar design discussion I had with Mark yesterday. We're evolving a new plan for toolbars in MD apps, and I think we will avoid needing anything like this. This (and to a lesser extent `WithToolbar`) are stepping stones, porting the U1 functionality to U2. They're not the final state.